### DISCUSSION TITLE
vcpkg reports that (C++) zlib[core]:x86-windows is installed, but no package directory

### DISCUSSION THREAD

**Discussion by aiqsystems** [microsoft/vcpkg#29961] [General]
After install of grpc only the x64 zlib package is there in packages directory.  If I subsequently try to run **_vcpkg install zlib zlib:x86-windows_** it says it is already installed.
Same problem with abseil, c-ares, upd and re2.

The client I am attempting to write is being added to an existing 32 bit application.

Thank you!

**Comment by dg0yt** [comment]
The final install tree is in `installed/<triplet/`, not in `packages/`.

**Comment by aiqsystems** [comment]
I guess I missed that.  Thanks!
